CHANGELOG.md

```diff
@@ -3,6 +3,29 @@
 
 # Prism Changelog
 
+## 0.8.0
+
+### Nesting Improvements
+
+PowerShell has a 10 directory nesting limit for nested modules.In order to prvent this nesting limit, Prism now:
+
+* installs nested modules directly in the module directory instead of a "PSModules" directory.
+* moves modules out of the versioned directory where PowerShell installs them by default (unless a module depends on
+  multiple versions of the same module).
+
+Prism determines if it is installing nested modules by looking for a .psd1 or .psm1 file in the same directory as the
+prism.json file.
+
+### Changes
+
+* The "PSModulesDirectoryName" configuration option can no longer be a path.
+* The `prism install` command now only returns objects for modules that were actually installed.

```

Comment:
Any updates we can make to reduce the need to add `| Format-Table -Auto` in calls like `prism install | Format-Table -Auto`?

Reply:
I checked and we introduced that funtionality awhile ago, but it isn't documented in the changelog. We should only have to do that if a script includes output of multiple object types.

Reply:
Actually, according to release notes, this formatting has been in place since Prism 0.3.0. I'll investigate why we don't see tables in our output. I wonder if Prism is out of date on build servers?

Reply:
Prism 0.7.0 is installed on all build servers at the machine level, along with several old versions. I've uninstalled all the old versions. We should be able to remove all the ` | Format-Table` statements now.